src/tests/Interop/PInvoke/Attributes/SuppressGCTransition/SuppressGCTransitionTest.cs

```diff
@@ -8,36 +8,52 @@
 using System.Runtime.InteropServices;
 using TestLibrary;
 
-static class SuppressGCTransitionNative
+unsafe static class SuppressGCTransitionNative
 {
-    [DllImport(nameof(SuppressGCTransitionNative), EntryPoint = "NextUInt")]
+    [DllImport(nameof(SuppressGCTransitionNative), CallingConvention=CallingConvention.Cdecl, EntryPoint = "NextUInt")]
```

Comment:
Do we need CDECL for a reason?

Reply:
Yes, the only macros we have in native code for the same calling convention across platforms gives us cdecl and the IL component of this test is testing the old calli syntax which has one hard coded calling convention for all platforms.